static/app/components/events/interfaces/frame/context.spec.tsx

```diff
@@ -0,0 +1,52 @@
+import {Coverage, LineCoverage} from 'sentry/types';
+
+import {getCoverageColors} from './context';
+
+describe('Frame - Context', function () {
+  const org = TestStubs.Organization();
+  const lines = [
+    [231, 'this is line 231'],
+    [232, 'this is line 232'],
+    [233, 'this is line 233'],
+    [234, 'this is line 234'],
+  ] as Array<[number, string]>;
```

Comment:
prefer types to casting
```suggestion
  const lines: Array<[number, string]> = [
    [231, 'this is line 231'],
    [232, 'this is line 232'],
    [233, 'this is line 233'],
    [234, 'this is line 234'],
  ];
```